### DISCUSSION TITLE
Handling of QSL cards images

### DISCUSSION THREAD

**Discussion by dl2ki** [foldynl/QLog#239] [General]
In the window "Sync & QSL, Paper QSL", a default directory "/home/xxx/.local/share/hamradio/QLog" is given. If you now press "Browse", the selection window opens in the QLog start directory. In the window "Sync & QSL, eQSL" this is also the case.

Maybe it would make more sense if pressing "Browse" would open the dialog in the directory already entered in "Save QSLs to". But for this there can be also several views.

Another question is whether the description "Save QSLs to" is correct here. This would only make sense if you could save paper QSL cards directly from QLog.

The way I see it, you can scan paper QSL cards, and put the images in that directory. Instead of "Save QSLs to" maybe "Paper QSL Directory" or similar would be more suitable.

In the "QSO Detail" window, the "Show QSL Card" button is grayed out. The button only works via eQSL if "eQSL_r = Y", but then downloads the "QSL Card" online, apparently without using the directory specified under "Sync & QSL, eQSL". Why are the eQSL images stored locally at all.

A "QSL Card" from one of the "Save directories" can only be displayed via the "Manage QSL Card" dialog, if the image has been stored there before. It would also only work with one file, e.g. the "QSL front Picture".

I'm not sure I understand the strategy behind managing the "QSL card images". However, this may be because I have never used these options in a logbook program myself. On the other hand, it might be useful to think about whether there is a need for change or unification here.

So I'll just put my thoughts on this here for discussion.

**Comment by foldynl** [comment]
> In the window "Sync & QSL, Paper QSL", a default directory "/home/xxx/.local/share/hamradio/QLog" is given. If you now press "Browse", the selection window opens in the QLog start directory. In the window "Sync & QSL, eQSL" this is also the case.
> 
> Maybe it would make more sense if pressing "Browse" would open the dialog in the directory already entered in "Save QSLs to". But for this there can be also several views.

Yes...it makes sense. it will be changed


> Another question is whether the description "Save QSLs to" is correct here. This would only make sense if you could save paper QSL cards directly from QLog.
> 
> The way I see it, you can scan paper QSL cards, and put the images in that directory. Instead of "Save QSLs to" maybe "Paper QSL Directory" or similar would be more suitable.

> I'm not sure I understand the strategy behind managing the "QSL card images". However, this may be because I have never used these options in a logbook program myself. On the other hand, it might be useful to think about whether there is a need for change or unification here.

As you wrote, the word "SAVE" is probably not the right term, because QLog "IMPORTS" the QSL image. This means that QLog will take images from the original location and "copy" it to the target directory. That's why there is a QLog option in Settings where the QSL images should be saved. Why is this happening? This is due to the fact that QLog prevents images from being "lost" from their original location. This can happen because the operator renames the file or deletes it with the help of another application.

I'll try to reword it somehow so that it makes sense in the next QLog version.

> In the "QSO Detail" window, the "Show QSL Card" button is grayed out. The button only works via eQSL if "eQSL_r = Y", but then downloads the "QSL Card" online, apparently without using the directory specified under "Sync & QSL, eQSL". Why are the eQSL images stored locally at all.

It seems that it must be reworded too. Show QSL Card is related only to eQSL service. On the other hand, Manage QSL Card is related to the "Paper QSL" only.

**Comment by dl2ki** [comment]
Thank you for the additional explanations, which are helpful in understanding the internal relationships in this context.  I think that the choice of appropriate terms will also generally contribute to a better understanding of the functionality.

According to your explanations, I will try this once. But a use of these features is not planned here.

**Comment by foldynl** [comment]
Updated based on your comments. Hopefully it will be better now. Thanks
